src/index.ts

```diff
@@ -18,3 +20,26 @@ export async function run(options: {
     () => { }
   );
 }
+
+/**
+ * Transform JS source (compiled output of `elm make`)
+ * Handy for making build tool plugins (e.g, parcel, snowpack, webpack, etc.)
+ */
+export async function transform(
```

Comment:
Why is the function async and why does it return a promise? 🤔

Reply:
Good question. There doesn't seem to be a need for `Transform.transform` to be async. I've updated my code as well as the internal `Transform.transform` to no longer be async.

Nice find.